src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Comment } from '../type/comment';
+import './CommentInfo.scss';
+
+type Props = {
+  comment: Comment
+};
+
+export const CommentInfo: React.FC<Props> = ({ comment }) => (
```

Comment:
I think you could not create a new type and just use Comment here
```suggestion
export const CommentInfo: React.FC<Comment> = ({ comment }) => (
```